keyboards/handwired/justageng/office60/info.json

```
@@ -0,0 +1,78 @@
+{
+    "keyboard_name" : "office60",
+    "bootloader": "stm32duino",
+    "processor": "STM32F103",
+    "url": "https://github.com/JiaGengChang/qmk_firmware",
+    "maintainer": "JiaGengChang",
```

0x0FF1 is in use:
https://www.the-sz.com/products/usbid/index.php?v=0x0FF1&p=&n=

In general, the further past 0x4000 the VID is, the better.  I've set it to FEED, for the meanwhile. 

```suggestion
    "maintainer": "JiaGengChang",
    "usb": {
        "vid": "0xFEED",
        "pid": "0xCE60",
        "device_version": "0.0.1"
    },
```